walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/MeetingRanges.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Stack;
+
+/**
+ * Class representing a collection of Intervals where a meeting could be scheduled. 
+ */
+public class MeetingRanges {
+
+  private static HashSet<Interval> intervals;
```

Comment:
This shouldn't be `static` because it's redefined for each instance of the class.